processor/schematransformerprocessor/README.md

```diff
@@ -0,0 +1,25 @@
+# Schema Transformer Processor
+
+🚧 _Currently under development, subject to change_ 🚧
+
+Supported Pipelines: traces, metrics, logs
+
+The _Schema Transformer Processor_ is used to convert existing telemetry data or signals to the most recent targeted version of the semantic convention defined as part of the configuration. The latest schema translation file will be bundled as part of the collector build to avoiding having to fetch it at runtime. 
+
+Specification Reference: [here](https://opentelemetry.io/docs/reference/specification/schemas/overview/)
+
+
+# Schema Definitions
+All of the published schema transformation are found by `https://opentelemetry.io/schemas/`
+
+# Example
+
+```yaml
+processors:
+    schema_transformer:
```

Comment:
Can we simplify the name to `schema`? Is there anything else that we can do with the schema other than transform it?

Reply:
For this, I was following what was written into the spec however, the spec itself uses two seperate names for the component so I guess it shouldn't matter too much.